app/controllers/items/detail.js

```diff
@@ -624,14 +665,36 @@ export default GoodcityController.extend(
           .then(data => {
             if (data.length) {
               this.set("containedPackages", data);
+
+              // Merge with the existing records
+              // Update the existing records with new record
+              let containedPackages = this.get("containedPackages");
+              containedPackages = _.unionBy(data, containedPackages, "id");
+
+              this.set("containedPackages", containedPackages);
             }
+            return data;
           });
       },
 
       openItemsSearch() {
         this.set("openPackageSearch", true);
       },
 
+      async updateAssociatedPackages(pkg) {
```

Comment:
I think the terminology `associated packages` is confusing. Should always be either `contained packages` or `container`